sdk/metricsadvisor/ai-metrics-advisor/samples/v1/javascript/README.md

```diff
@@ -0,0 +1,82 @@
+| [ingestionStatus.js][ingestionstatus]       | This sample demonstrates operations related to ingestion status.                                                                                                                       |
+| [metricFeedback.js][metricfeedback]         | This sample demonstrates how to provide feedback for a metric.                                                                                                                         |
+| [metricQueries.js][metricqueries]           | This sample demonstrates querying methods related to metric.                                                                                                                           |
+| [quickstart.js][quickstart]                 | This sample demonstrates how to get started by creating a data feed, checking ingestion status, creating detection and alerting configurations, and querying for alerts and anomalies. |
+| [seriesData.js][seriesdata]                 | This sample demonstrates how to retrieve time series data.                                                                                                                             |
+
+## Prerequisites
+
+The sample programs are compatible with Node.js >=12.0.0.
+
+You need [an Azure subscription][freesub] and the following Azure resources to run these sample programs:
+
+- [Azure Cognitive Services account][createinstance_azurecognitiveservicesaccount]
+
+Samples retrieve credentials to access the service endpoint from environment variables. Alternatively, edit the source code to include the appropriate credentials. See each individual sample for details on which environment variables/credentials it requires to function.
+
+Adapting the samples to run in the browser may require some additional consideration. For details, please see the [package README][package].
+
+## Setup
+
+To run the samples using the published version of the package:
+
+1. Install the dependencies using `npm`:
+
+```bash
+npm install
+```
+
+2. Edit the file `sample.env`, adding the correct credentials to access the Azure service and run the samples. Then rename the file from `sample.env` to just `.env`. The sample programs will read this file automatically.
+
+3. Run whichever samples you like (note that some samples may require additional setup, see the table above):
+
+```bash
+node alertingConfig.js
+```
+
+Alternatively, run a single sample with the correct environment variables set (setting up the `.env` file is not required if you do this), for example (cross-platform):
+
+```bash
+npx cross-env METRICS_ADVISOR_ENDPOINT="<metrics advisor endpoint>" METRICS_ADVISOR_SUBSCRIPTION_KEY="<metrics advisor subscription key>" METRICS_ADVISOR_API_KEY="<metrics advisor api key>" METRICS_ADVISOR_DETECTION_CONFIG_ID="<metrics advisor detection config id>" METRICS_ADVISOR_HOOK_ID="<metrics advisor hook id>" node alertingConfig.js
```

Comment:
Are there any samples that are simpler or don't require as many environment variables that you could make be the first sample in the list? This ends up looking a little messy because this sample uses so many vars.

Reply:
The simplest sample is just the authentication part and listing anything (like datafeeds say). Even in that atleast 4 environment vars will be required though

Reply:
I can use quickstart.ts as the first sample. It has 5 env vars but covers all the basic scenarios.